iocBoot/iocEK9KTest/gen-cmd.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+"""
+Generate st.cmd from a list of terminals, or the coupler's modbus configuration
+"""
+
+import os
+import sys
+import re
+import argparse
+
+use_modbus = True
+try:
+	from pymodbus.client import ModbusTcpClient
+except:
+	print('Failed to import pymodbus, terminals must be manually specified on the command line')
+	use_modbus = False
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--ip', type=str, required=True, help='IP address and port the coupler can be reached at')
+parser.add_argument('--out', type=str, default='st.cmd', help='File to output to')
+parser.add_argument('--terminals', type=str, help='Comma separated list of terminals')
+parser.add_argument('--port', type=int, default=502, help='Port to use')
+args = parser.parse_args()
+
+os.chdir(os.path.dirname(__file__))
+
+arch = os.getenv('EPICS_HOST_ARCH')
+if arch is None:
+	print('EPICS_HOST_ARCH environment variable not set')
+	exit(1)
```

Comment:
I learned (re-learned? my memory is faulty) something new today from this: `exit()` is a built-in.
Technically you should be using `sys.exit` here, but both work so it's not a big deal to change.

ref: https://stackoverflow.com/questions/6501121/difference-between-exit-and-sys-exit-in-python